README.md

```diff
@@ -4,7 +4,7 @@
 This API client was generated by the [OpenAPI Generator](https://openapi-generator.tech) project.  By using the [OpenAPI-spec](https://www.openapis.org/) from a remote server, you can easily generate an API client.
 
 - API version: 2023-04-10
-- Package version: 1.1.0
+- Package version: 1.0.0
```

Comment:
This looks like its reverting the version

Reply:
Hmm, I see, this is being generated from something that I didn't know about. I changed it manually to 1.1.0.  Do you know what its updated from?  It would be good to fix that :-)